include/lemlib/chassis/differential.hpp

```diff
@@ -302,8 +304,8 @@ class Differential : public Chassis {
          */
         void update() override;
 
-        ControllerSettings linearSettings;
-        ControllerSettings angularSettings;
+        struct ControllerSettings<Length> lateralSettings;
```

Comment:
```suggestion
        struct ControllerSettings<Length> linearSettings;
```
rename lateral to linear